### PR TITLE
feat: action to create issue on english changes

### PR DESCRIPTION
**Pull request by moT01**
This adds an action to create an issue when english text changes. It can serve as a temp way to make sure the translations are kept in sync with english until we move the files to crowdin.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Comment by nhcarrigan** [comment]
So to clarify, if I add a new key to the `translations.json` object the flow would be:

- Add the key-value pair in english to `english/translations.json`.
  - Add that same key-value pair in english to all other languages' `translations.json` and the schema file (or the lint CI will fail)
- PR gets merged, this action fires to create an issue to indicate those new keys need translations

Right?

**Comment by RandellDawson** [comment]
Why don't we just upload it to Crowdin now instead?  We could have the file uploaded to Crowdin if committed to `master`.

So we don't have so many different actions firing related to changes to files, why not combine all such actions that look at commits to `master` on specific files into a single action.  @nhcarrigan already has one that prevents a PR from being merged if non-English curriculum files appear on the PR.  We probably need to discuss all the things we want to check and then create a single action.

Actually, we could add your JSON files to the existing upload workflow that we are already using for Crowdin translations.

**Comment by nhcarrigan** [comment]
> So we don't have so many different actions firing related to changes to files, why not combine all such actions that look at commits to master on specific files into a single action.

The catch to this is, after some testing, I did determine that you cannot have specific `path` options for sections of a workflow. It's only a top level configuration.

**Comment by RandellDawson** [comment]
What I am really suggesting is to not use this action for these files.  We can move the `curriculum/crowdin.yml` file to `tools/crowdin` and add the following to the file:

We will be setting this up to upload English source files, so why not add it to this file?  On Crowdin, we can go in an set a higher priority for the translations to these files.

**Comment by moT01** [comment]
yes, @nhcarrigan - if any changes are made to those two files listed, an issue would be created. 

This action could be removed once we get the files on crowdin. I thought that, for now, this would just keep track of changes - really it should be looking at PR's going to the next branch until that gets merged into master.

**Comment by raisedadead** [comment]
I think we do not need to add these files on master, or upload them right now. We really do not need to worry about keeping things in sync from branch to branch.

As I understand the files like `translations.json`, `intro.json` & anything else will just land into master when next is merged. We can update the config (the existing action) to upload these new files. We will additionally upload the translated files for spanish and chinese and get them re-approved on Crowdin auditing and fixing any bits that have not yet been translated.

In short I do not think we need to track or seed any files right now.
